pdsfile/pdsfile.py

```
@@ -3703,15 +3679,15 @@ def child(self, basename, fix_case=True, must_exist=False,
+                    this.bundletype_   = matchobj.group(3).lower() + '/'
                 else:
                     this.checksums_ = matchobj.group(1)
                     this.archives_  = matchobj.group(2)
-                    this.voltype_   = matchobj.group(3) + '/'
+                    this.bundletype_   = matchobj.group(3) + '/'
 
-                if this.voltype_[:-1] not in cls.VOLTYPES:
+                if this.bundletype_[:-1] not in cls.VOLTYPES:
                     raise ValueError('Unrecognized volume type "%s": %s' %
-                                     (this.voltype_[:-1], this.logical_path))
+                                     (this.bundletype_[:-1], this.logical_path))
```

Ruff (0.13.3)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)